Formula/mkvtoolnix.rb

```diff
@@ -3,7 +3,7 @@ class Mkvtoolnix < Formula
   homepage "https://mkvtoolnix.download/"
   url "https://mkvtoolnix.download/sources/mkvtoolnix-30.1.0.tar.xz"
   sha256 "4628f40d62d359edb1441c52381b1f3a61aa227279133d7e01257f91e0d92591"
-  revision 2
+  revision 3
```

Comment:
Please split this into a separated commit, with commit message like `mkvtoolnix: revision for libmatroska`, as on the one-formula-per-commit and one-commit-per-formula basis, thanks!